pkg/controller/direct/orgpolicy/policy_fuzzer.go

```diff
@@ -42,13 +42,6 @@ func orgPolicyPolicyFuzzer() fuzztesting.KRMFuzzer {
 	f.UnimplementedFields.Insert(".name")
 	f.UnimplementedFields.Insert(".alternate") // deprecated field
 	f.UnimplementedFields.Insert(".dry_run_spec.etag")
```

Comment:
The related functions are `PolicySpec_PolicyRule_FromProto` and `PolicySpec_PolicyRule_ToProto` in  pkg/controller/direct/orgpolicy/policy_mapper.go